src/layouts/ReviewRequest/Dashboard.tsx

```diff
@@ -13,127 +13,271 @@ import {
+
+import { SiteViewHeader } from "../layouts/SiteViewLayout/SiteViewHeader"
+
+import { CancelRequestModal, ManageReviewerModal } from "./components"
+import { ApprovedModal } from "./components/ApprovedModal"
+import { PublishedModal } from "./components/PublishedModal"
+import { RequestOverview } from "./components/RequestOverview"
+
+export const ReviewRequestDashboard = (): JSX.Element => {
+  const { role } = useRoleContext()
+  const { siteName, reviewId } = useParams<{
+    siteName: string
+    reviewId: string
+  }>()
+  const { setRedirectToPage } = useRedirectHook()
+  // TODO!: Refactor so that loading is not a concern here
+  const { data, isLoading: isGetReviewRequestLoading } = useGetReviewRequest(
+    siteName,
+    parseInt(reviewId, 10)
+  )
+  const { data: collaborators, isLoading, isError } = useGetCollaborators(
+    siteName
+  )
+  // TODO!: redirect to /sites if cannot parse reviewId as string
+  const { onCopy, hasCopied } = useClipboard(data?.reviewUrl || "")
+
+  const reviewStatus = data?.status
+  const isApproved = reviewStatus === ReviewRequestStatus.APPROVED
+
+  useEffect(() => {
+    if (
+      reviewStatus === ReviewRequestStatus.CLOSED ||
+      reviewStatus === ReviewRequestStatus.MERGED
+    ) {
+      setRedirectToPage(`/sites/${siteName}/dashboard`)
+    }
+  }, [reviewStatus, setRedirectToPage, siteName])
+
+  return (
+    <>
+      <Box bg="white" w="100%" h="100vh">
+        <VStack
+          bg="blue.50"
+          pl="9.25rem"
+          pr="2rem"
+          pt="2rem"
+          pb="1.5rem"
+          spacing="0.75rem"
+          align="flex-start"
+        >
+          <Flex w="100%">
+            <HStack spacing="0.25rem">
+              <Text textStyle="h5">{data?.title || ""}</Text>
+              {/* Closes after 1.5s and does not refocus on the button to avoid the outline */}
+              <Popover returnFocusOnClose={false} isOpen={hasCopied}>
+                <PopoverTrigger>
+                  <IconButton
+                    icon={<BiLink />}
+                    variant="clear"
+                    aria-label="link to pull request"
+                    onClick={onCopy}
+                  />
+                </PopoverTrigger>
+                <PopoverContent
+                  bg="background.action.alt"
+                  _focus={{
+                    boxShadow: "none",
+                  }}
+                  w="fit-content"
+                >
+                  <PopoverArrow bg="background.action.alt" />
+                  <PopoverBody>
+                    <Text textStyle="body-2" color="text.inverse">
+                      Link copied!
+                    </Text>
+                  </PopoverBody>
+                </PopoverContent>
+              </Popover>
+            </HStack>
+            <Spacer />
+            <CancelRequestButton isApproved={isApproved} />
+            <ApprovalButton isApproved={isApproved} />
```

Comment:
For some reason these two buttons appear the same on my side:

<img width="356" alt="Screenshot 2022-10-18 at 10 11 10" src="https://user-images.githubusercontent.com/27919917/196319077-7785c98a-cec9-4bad-babf-338e6827de32.png">

I think maybe need to check the reviewer role to display the correct button, instead of disabling?

Reply:
good catch, this change was made for manual testing and i missed out on reverting